src/config-sample.php

```diff
@@ -86,13 +86,6 @@
      */
     'disable_auto_cron' => false,
 
-    /*
-     * Enable or disable search engine friendly URLs.
-     * Configure .htaccess file before enabling this feature
-     * Set to TRUE if using nginx.
-     */
-    'sef_urls' => true,
```

Comment:
When have we decided to drop support for it?

Reply:
This was discussed on Discord, and I was advised that it's currently broken and it would be best to remove support - is this not the case?